### DISCUSSION TITLE
Welcome to Windrecorder Discussions! 欢迎来到捕风记录仪的讨论板块！

### DISCUSSION THREAD

**Discussion by Antonoko** [Antonoko/Windrecorder#63] [Announcements 公告]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

## 👋 欢迎！
   我们使用讨论作为与社区其他成员联系的地方。 我们希望您：
   * 提出您想知道的问题。
   * 分享意见。
   * 与其他社区成员互动。
   * 欢迎他人，思想开放。 请记住，这是一个我们共同建设的社区。

   首先，请在下面发表评论并介绍一下您自己，并告诉我们您在这个社区中所做的事情。

**Comment by Antonoko** [comment]
> [!NOTE]
> 这里是 Windrecorder 的讨论区，欢迎一起讨论一切🪼

我是这个工具的最初发起者，完成了最初版本的搭建。由于专业技能并不以程序见长，目前主要负责从产品、设计等职能维护这个工具。这个项目也是我学习代码的实践过程，如果有遇到使用问题与 Bug，欢迎提交 issue 与 PR 一起共建！

目前较为常驻、参与贡献这个业余项目的才华横溢成员有：

@hatsuroku ：完成了大量代码的重构与调优，解决许多顽疾 Bug，使项目更加科学规范、易于开发；

@jpswing ：完成了大量代码的重构调优；目前作为 pr 的 code reviewer；

~@ShellBin ：正在开发一些前端特性的新 features；~

……

受限于开发者们业余有限的时间精力，我们可能无法及时地响应与修缮更新项目，敬请理解。如果对此项目有兴趣，欢迎一起参与贡献到这个项目中来，**你可以在此留言介绍自己、所擅长的、所好奇的、所想做的一切🪼**

同时欢迎在讨论区的各个板块分享你的想法！

---

> [!NOTE]
> This is the discussion forum of Windrecorder, welcome to discuss everything together🪼

I was the original initiator of this tool and completed the construction of the initial version. Since my professional skills are not focused on programming, I am currently mainly responsible for maintaining this tool from product, design and other functions. This project is also my practical process of learning code. If you encounter any usage problems or bugs, please feel free to submit issues and PRs to build together!

The talented members who are currently resident and contributing to this side project include:

@hatsuroku: Completed a large amount of code reconstruction and tuning, solved many stubborn bugs, and made the project more scientific, standardized and easy for development;

@jpswing: Completed a lot of code refactoring and tuning, and are developing new features to make the tool easier to use;

@ShellBin: Some new features of front-end features are being developed;

…

Due to the limited time and energy of developers, we may not be able to respond and repair and update projects in a timely manner, so please understand. If you are interested in this project, you are welcome to participate and contribute to this project. **You can leave a message here to introduce yourself, what you are good at, what you are curious about, and everything you want to do🪼**

You are also welcome to share your thoughts in various sections of the discussion forum!

**Reply by ASC8384** [reply]
It's so cool, I want to be the maintainer.
I developed three features, more compression options, start of the day, and multiple screen recording support.
~~Though they were all greatly helped by LZ (and didn't merge)~~

**Comment by Antonoko** [comment]
> It's so cool, I want to be the maintainer. I developed three features, more compression options, start of the day, and multiple screen recording support. ~Though they were all greatly helped by LZ (and didn't merge)~

I've added you as a project collaborator, take a moment to introduce yourself to everyone so we can get to know you better🪼

**Reply by ASC8384** [reply]
I am currently engaged in LLM related work, and can help with the migration of software to Linux and MacOS.

**Reply by Antonoko** [reply]
sounds amazing🤯 pls feel free to share your plan or specific thought on issue so we can discuss further more (i am exploring LLM application right now too lol
（my previously shit code might too coupled with Windows, could be hard work to refactor、

**Comment by B1lli** [comment]
这个项目真的很出色，希望能看到更多更快的进展，我也想成为项目的合作开发者，可以一起合作开发OCR相关的功能，或者帮忙打包安装程序，此外，如果@Antonoko 愿意的话，我也希望能将项目宣传给更多有需要的人

**Reply by Antonoko** [reply]
谢谢喵！如果能随意推广宣传我也会很开心 xd
如果有想要参与开发的部分，欢迎在 issue 中新开一个帖子一起详细讨论，开发时开一个分支 pr 就可以了。之后如果需要什么 contributor 权限上帮助我可以把你加上权限角色
（太苦了社畜😭希望能抽更多空闲干活）

**Reply by B1lli** [reply]
我的产品（copyask.com）官网每天有蛮多流量的，如果你不介意，我想把捕风记录仪挂上去帮忙宣发，标明开发者是你，具体我们可以私信说

**Reply by Antonoko** [reply]
看上去很棒！不过这个小工具我还是希望保持比较独立与随性，看上去调性不大适合放上去宣传hh，谢谢好心。我的邮箱是 antonoko@outlook.com，有其他需要可以随时联系。
